sumologic/resource_sumologic_generic_polling_source.go

```diff
@@ -130,6 +130,22 @@ func resourceSumologicGenericPollingSource() *schema.Resource {
 						},
 					},
 				},
+				"sns_topic_or_subscription_arn": {
```

Comment:
Can we update the documentation as how to `sns_topic_or_subscription_arn` field is expected to be used? i.e. create -> update state outside of terraform -> refresh state.

Reply:
added